resources/upload.js

```diff
@@ -0,0 +1,30 @@
+const { CosmosClient } = require('@azure/cosmos')
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
+
+const endpoint = 'https://localhost:8081'
+const key = 'C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw=='
+const messagesDatabase = process.argv[4]
+const messagesContainer = process.argv[5]
+const client = new CosmosClient({ endpoint, key })
+
+
+const uploadData = async () => {
+  for (let i = 0; i < process.argv[6]; i++) {
+    const item = {
+      scheme: 'AHWP',
+      tags: ['tag 1', 'tag 2'],
+      crn: `${process.argv[2]}`,
+      sbi: `${process.argv[3]}`,
```

Comment:
Do the arguments for `crn` and `sbi` above (`${process.argv[2]}` and `${process.argv[2]}` respectively) just refer to whatever the input value was in the `cosmos-upload` script?

Reply:
Yes thats correct, `process.argv` gives access to all the arguments passed after calling `node resources/upload.js`. 

The first two elements of the array are included by default `process.argv[0]` and `process.argv[1]`. You can read about them [here](https://nodejs.org/docs/latest/api/process.html#processargv). 

So the first argument that is passed in our script is `process.argv[2]` which I have set as the crn.

Reply:
Ah I see. Cool! Thanks for the explanation.